README.md

```diff
@@ -2,19 +2,29 @@
 
 ## Description
```

Comment:
Here's some more questions to answer in the description:
- What was your motivation?
- Why did you build this project? (Note: the answer is not "Because it was a homework assignment.")
- What problem does it solve?
- What did you learn?